### PR TITLE
Update of-ac-test config

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/buyingcatalogue/templates/configmap.yaml [hunk 1]
```diff
@@ -41,4 +41,5 @@ data:
   document-container: {{ $container | quote }}
   run-id: {{ $runId | quote }}
   mp-supplier-url: {{ printf "%s/marketing/supplier/solution/" $pbUrl }}
-  mp-authority-url: {{ printf "%s/marketing/authority/solution/" $pbUrl }}
\ No newline at end of file
+  mp-authority-url: {{ printf "%s/marketing/authority/solution/" $pbUrl }}
+  of-ac-url: {{ printf "%s/order" $pbUrl }}
```
Comment (gavinwakeley1) [inline src/buyingcatalogue/templates/configmap.yaml:45]: Should this be "%s/order" or "%s/order/" - not sure if it's significant, but to be consistent

**Review by gavinwakeley1 — COMMENTED** [review]
Looks good otherwise - can you confirm answer to comment and I'll approve

File: src/buyingcatalogue/templates/configmap.yaml (hunk 1 above)
Comment (PhysicsJon) [reply src/buyingcatalogue/templates/configmap.yaml:45]: I think it should have the trailing `/`, I'll update

**Review by gavinwakeley1 — APPROVED** [review]
Looks good

**Pull request merged**